webui/src/lib/api/index.js

```diff
@@ -71,6 +71,9 @@ const apiRequest = async (uri, requestData = {}, additionalHeaders = {}) => {
             cache.delete('user');
             throw new AuthenticationError('Authentication Error', response.status);
         }
+        if (errorMessage === '') {
+            throw new AuthorizationError("Insufficient Permissions", response.status);
+        }
         throw new AuthorizationError(errorMessage, response.status);
```

Comment:
I think this might be a bit more readable: it identifies the "unauthorized" case separately, and then handles it in one way.  Also it will ensure an error message if something even stranger happens in extractError and e.g. `errorMessage === undefined`.

```suggestion
        throw new AuthorizationError(errorMessage || 'Insufficient Permissions', response.status);
```

(This also fixes a nit, that other strings nearby all use single quotes)

Reply:
Sounds great, lets go for it